pyproject.toml

```diff
@@ -28,6 +28,7 @@ boto3 = ">=1.16.13"
 botocore = ">=1.19.13"
 flask-caching = "*"
 flask-unleash = "^2.0.0"
+k8s-deployment-status = "^0.0.6"
```

Comment:
You can add this package in the [Github action](https://github.com/RedHatInsights/ros-backend/blob/main/.github/workflows/build_with_latest_direct_deps.yml) file for direct dependency as tests are failing for it.